RELEASE/relay/relay_autoscend.ash

```diff
@@ -51,6 +51,13 @@ void handleSetting(string type, int x)
 	}
 
 	setting set = s[type][x];
+
+	// Some settings we don't want to appear in the big table, as they're used elsewhere
+	if ((set.name == "_auto_doneToday_confirm") || (set.name == "_auto_doneToday"))
+	{
+		return;
+	}
```

Comment:
The point of the built settings is specifically for them to show up in relay. I don't see the logic of first adding it to the relay data table, then adding code into the relay script to hide it.

If you don't want it to show up then just delete it from [action.dat]

Also. Why wouldn't we want to see it? I think it is worth leaving it so we can see it. as such I would suggest removing this bit

Reply:
I was somehow under the impression that relay_autoscend.ash would ignore properties that were not included in autoscend_settings.txt.  Playing around a bit, I see I was wrong.  I'll take those properties out.